lib/backend.js

```diff
@@ -12,18 +12,18 @@ const defaults = {
 	prefix : ''
 };
 
-function SequelizeBackend (db, options){
+function SequelizeBackend (db, options, Sequelize){
 	this.options = _.defaults({}, options, defaults);
 	this.db = dbTasks.setup(db, this.options);
-	this.Op = this.db.Op || {in: '$in'};
-	this.Promise = this.db.Promise;
+	this.Op = Sequelize.Op || {in: '$in'};
+	this.Promise = Sequelize.Promise;
```

Comment:
We can try getting  `Op` and `Promise` from the class and if it fails get it from the connection object. That way we keep compatibility with v4 and v3 -
`this.Op = this.db.Sequelize.Op || this.db.Op || {in: '$in'};`